test/integration/pipeline/tx_pipeline_integration_test.cpp

```diff
@@ -165,16 +174,19 @@ TEST(PipelineIntegrationTest, SendQueryWithValidation) {
  * @then receive STATELESS_VALIDATION_SUCCESS status on that tx
  */
 TEST(PipelineIntegrationTest, SendTx) {
-  iroha::model::generators::CommandGenerator gen;
-
-  iroha::model::Transaction tx;
-  tx.created_ts = iroha::time::now();
-  tx.commands.push_back(gen.generateAddAssetQuantity(
-      "user", "test", iroha::Amount().createFromString("0").value()));
+  auto tx = shared_model::proto::TransactionBuilder()
+                .createdTime(iroha::time::now())
+                .creatorAccountId(kUser)
+                .txCounter(1)
+                .addAssetQuantity(kUser, kAsset, "1.0")
+                .build()
+                .signAndAddSignature(
+                    shared_model::crypto::DefaultCryptoAlgorithmType::
+                        generateKeypair());
 
   auto check = [](auto status) {
     ASSERT_EQ(iroha::model::TransactionResponse::STATELESS_VALIDATION_SUCCESS,
-              status);
+              status.oldModelStatus());
```

Comment:
```C++
auto check = [](auto status) {
    ASSERT_NO_THROW(
        boost::get<shared_model::detail::PolymorphicWrapper<
            shared_model::interface::StatelessValidTxResponse>>(status.get()));
  };
```
Since we are removing old model.